features/convert_config.feature

```diff
@@ -0,0 +1,381 @@
+Feature: Convert config
+  In order to migrate the existing yaml configuration
+  As a developer
+  I need to be able to convert this configuration to the new PHP format
+
+  Background:
+    Given I set the working directory to the "ConvertConfig" fixtures folder
+    And I clear the default behat options
+    And I provide the following options for all behat invocations:
+      | option           | value |
+      | --no-colors      |       |
+      | --convert-config |       |
```

Comment:
In all these cases we are calling the new --convert-config option and use the --config option to specify which file to convert. Previously we copy one or more files to be converted to the temp folder and then we check the converted file